charts/core/config/log4j2.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN" monitorInterval="10">
```

Comment:
I had this in the log4j2.xml for docker so devs can change how log4j2 logs without needing to restart DHIS2.

https://logging.apache.org/log4j/2.x/manual/configuration.html#automatic-reconfiguration

Does this make sense for us in k8s? Would we want to update the configmap of a running instance so it picks up a changes in the log4j2 config 🤔